src/main/interface/cli.c

```diff
@@ -3360,91 +3360,101 @@ static void cliVersion(char *cmdline)
 typedef struct {
     const uint8_t owner;
     pgn_t pgn;
+    uint16_t stride;
     uint16_t offset;
     const uint8_t maxIndex;
 } cliResourceValue_t;
 
+#define DEFS(owner, pgn, type, member, max) \
```

Comment:
Maybe comment what does the macro do (I assume `S`ingle/`M`ultiple?)

Maybe another macro can be used for single-pin entris (maxIndex == 0)